src/test/resources/generator-tooltip-data.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id" : "GEN",
+  "p" : 25.0,
+  "q" : 32.0,
+  "targetP" : 28.0,
+  "targetQ" : 301.0,
+  "targetV" : 24.5,
```

Comment:
TargetV does not seem to be used in frontend ? Same for name ?
To remove if not used